src/ceph-disk/ceph_disk/main.py

```diff
@@ -3153,6 +3153,61 @@ def move_mount(
     )
 
 
+def systemd_enable(
+    path,
+    osd_id,
+):
+    # ensure there is no duplicate ceph-osd@.service
+    for style in ([], ['--runtime']):
+        command_check_call(
+            [
+                'systemctl',
+                'disable',
+                'ceph-osd@{osd_id}'.format(osd_id=osd_id),
+            ] + style,
```

Comment:
> --runtime
> When used with enable, disable, edit, (and related commands), make changes only temporarily, so that they are lost on the next reboot. This will have the effect that changes are not made in subdirectories of /etc but in /run, with identical immediate effects, however, since the latter is lost on reboot, the changes are lost too.

so if we want to disable the ceph-osd@<osd_id> unit permanently, we can just use `systemdctl disable ceph-osd@<osd_id>` without `--runtime`, right? and i am not sure why we need to disable and then enable then unit just for enabling it.

see https://www.freedesktop.org/software/systemd/man/systemctl.html